src/abstract/index.ts

```diff
@@ -1,4 +1,4 @@
-export { default as Model } from './Model';
-export { default as Collection } from './Collection';
-export { default as View } from './View';
-export { default as Module } from './moduleLegacy';
+export { default as Model } from "./Model";
+export { default as Collection } from "./Collection";
+export { default as View } from "./View";
+export { default as Module } from "./Module";
```

Comment:
Out of curiosity, any reason for why switching from single quotes to double quotes? 
As I don't see any change on the eslint config file, are you doing that change manually?

Reply:
So I think my editor do this I am not sure where its gets the config

Reply:
😬  are you able to fix it as it brings a bit of inconsistency?